joint_trajectory_controller/test/joint_trajectory_controller_vel.test

```diff
@@ -34,6 +34,6 @@
   <!-- Controller test -->
   <test test-name="joint_trajectory_controller_vel_test"
         pkg="joint_trajectory_controller"
-        type="joint_trajectory_controller_test"
-        time-limit="85.0"/>
+        type="joint_trajectory_controller_vel_test"
+        time-limit="120.0"/>
```

Comment:
Is this necessary?

Reply:
This is "rebase" on develop, picked the change from there to avoid merge conflicts.